src/current/cockroachcloud/metrics-sql.md

```diff
@@ -0,0 +1,61 @@
+---
+title: SQL metrics
+summary: The SQL metrics let you monitor SQL performance.
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+These graphs are available for CockroachDB {{ site.data.products.standard }} deployments. For graphs available to CockroachDB {{ site.data.products.advanced }} or {{ site.data.products.basic }} deployments, refer to the [CockroachDB {{ site.data.products.cloud }} Console Metrics page]({% link cockroachcloud/metrics.md %}#cockroachdb-cloud-console-metrics-page).
+{{site.data.alerts.end}}
+
+The SQL metrics let you monitor SQL performance.
```

Comment:
'SQL performance' could be a link to https://www.cockroachlabs.com/docs/v23.2/make-queries-fast so users have a place to start research if they're not liking what they're seeing in these metrics